jmdaemon/setup.py

```diff
@@ -9,6 +9,8 @@
       author_email='',
       license='GPL',
       packages=['jmdaemon'],
-      install_requires=['txtorcon', 'cryptography==3.3.2', 'pyopenssl', 'libnacl', 'joinmarketbase==0.9.9dev'],
+      install_requires=['txtorcon==22.0.0', 'cryptography==3.3.2',
+                        'pyopenssl==21.0.0', 'libnacl==1.8.0',
```

Comment:
libnacl is clearly fine, we have already been using that latest release for a while, and it's clearly stable.